PNA.adoc

```diff
@@ -356,101 +238,97 @@ In this document we use the following naming conventions:
-|-------------|---------------|--------------------|
-|             | Processed     | Resulting          |
-| Description | next by       | packet(s)          |
-+:------------+:--------------+:-------------------+
-| packet from   | main, with    | Zero or more mirrored  |
-| network port  | direction     | packets, plus at most  |
-|---------------|               | |
-| packet from   | `NET_TO_HOST` | one of: a net-to-host  |
-| net-to-host   |               | recirculated packet,   |
-| recirculate   |               | or one to-host packet. |
-|---------------|               |                        |
-| packet from   |               |                        |
-| port loopback |               |                        |
-|---------------|---------------|------------------------|
-| packet from   | main, with    | Zero or more mirrored  |
-| message processing | direction | packets, plus at most |
-|---------------|               | |
-| packet from   | `HOST_TO_NET` | one of: a host-to-net  |
-| host-to-net   |               | recirculated packet,   |
-| recirculate   |               | or one to-net packet.  |
-|---------------|               |                        |
-| packet from   |               |                        |
-| host loopback |               |                        |
-|---------------|---------------|------------------------|
-~
+
+.Result of packet processed one time by main
+[.center,%autowidth]
+[#results-of-one-pkt-thru-main]
+|===
+| *Description* |  *Processed next by*| *Resulting packet(s)*
+| packet from network port             1.3+^| main, with direction `NET_TO_HOST` 1.3+^|Zero or more mirrored packets, plus at most one of: a net-to-host recirculated packet, or one to-host packet.
+| packet from net-to-host recirculate 
+| packet from  port loopback
+| packet from  message processing      1.3+^| main, with  direction `HOST_TO_NET` 1.3+^|Zero or more mirrored packets, plus at most one of: a host-to-net recirculated packet, or one to-net packet.
+| packet from  host-to-net recirculate 
+| packet from  host loopback
+|===
 
 Note that each mirrored packet that results from `mirror_packet`
 operations will have its own next place that it will go to be
 processed, independent of the original packet, and independent of any
 other mirror copies made of the same original packet.
 
-# PNA Data types
+[#sec-pna-data-types]
+== PNA Data types
 
-## PNA type definitions {#sec-pna-type-definitions}
+[#sec-pna-type-definitions]
+=== PNA type definitions 
 
 Each PNA implementation will have specific bit widths in the data
 plane for the types shown in the code excerpt of Section
-[#sec-pna-type-definitions-code].  These widths are defined in the
+<<#sec-pna-type-definitions-code>>.  These widths are defined in the
 target specific `pna.p4` include file.  They are expected to differ
-from one PNA implementation to another[^PNAP4TargetSpecific].
+from one PNA implementation to another[1].
```

Comment:
I would recommend `[Footnote 2]` here, to avoid duplication with the earlier `[Footnote 1]`.